src/Behat/Behat/Tester/Cli/RerunController.php

```diff
@@ -156,12 +170,12 @@ private function generateKey(InputInterface $input)
     {
         return md5(
             $input->getParameterOption(array('--profile', '-p')) .
-            $input->getOption('suite') .
-            implode(' ', $input->getOption('name')) .
-            implode(' ', $input->getOption('tags')) .
-            $input->getOption('role') .
-            $input->getArgument('paths') .
-            $this->basepath
+                $input->getOption('suite') .
+                implode(' ', $input->getOption('name')) .
+                implode(' ', $input->getOption('tags')) .
+                $input->getOption('role') .
+                $input->getArgument('paths') .
+                $this->basepath
```

Comment:
I think this is just formatting change? Please revert if so

Reply:
Yeah, sorry. My formatting extension is kind of messed up.